src/text/classes.scss

```diff
@@ -1,35 +1,40 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
+@import './mixins';
 
-.iui-text-block {
-  @include iui-text-block;
+.iui-text-headline {
+  @include iui-text(headline);
 }
 
-.iui-text-headline {
-  @include iui-text-headline;
+.iui-text-title {
+  @include iui-text(title);
 }
 
-.iui-text-leading {
-  @include iui-text-leading;
+.iui-text-subheading {
+  @include iui-text(subheading);
 }
 
-.iui-text-muted {
-  @include iui-text-muted;
+.iui-text-leading {
+  @include iui-text(leading);
 }
 
-.iui-text-small {
-  @include iui-text-small;
+.iui-text-block {
+  @include iui-text(body);
 }
 
-.iui-text-subheading {
-  @include iui-text-subheading;
+.iui-text-small {
+  @include iui-text(small);
 }
 
-.iui-text-title {
-  @include iui-text-title;
+.iui-text-muted {
+  @include iui-text-muted;
 }
 
 .iui-skeleton {
   @include iui-text-skeleton;
 }
+
+.iui-text-spacing {
```

Comment:
I really like this cleanup! But I am afraid that we will break UIs for our users because when I used typography components, I haven't added any additional margin as they had it. Also, we now are adding spacing class to typography examples, so is it worth having spacing separate?

Reply:
So here's what I was thinking:
1. `.iui-text-spacing` would be applied to our current Typography components in React.
2. A new component called `Text` would not have spacing and support any kind of styling on any kind of element (e.g. h1 styling on h2).